src/Components/RequiredForm/RequiredForm.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import TextQuestion from '../TextQuestion/TextQuestion'
+import FormStartPage from '../FormStartPage/FormStartPage'
+import { gql, useMutation } from '@apollo/client';
+
+
+let RequiredForm = () =>{
+  let create_user = gql `
+   mutation createUser($name: String!, $email: String!, $birthdate: String!)
+  {
+    createUser(input: 
+    {
+      name: $name
+      email: $email
+      birthdate: $birthdate
+    })
+    {
+      id
+      name
+    }
+  }
+  `
+let [createUser, { data,loading,error,called }] = useMutation(create_user);
+
+let sendUser = async () =>{
+  
+  let Bday = answers[1]
+  
+  let finalBday = Bday.split('-').reverse().join('-')
```

Comment:
Do you think we should talk to backend about allowing any date format or keep this as is? Either way, nice error handling here for making sure the date is formatted correctly!